sktime/forecasting/base/_meta.py

```diff
@@ -25,9 +27,23 @@ class _HeterogenousEnsembleForecaster(_HeterogenousMetaEstimator, BaseForecaster
     # this must be an iterable of (name: str, estimator, ...) tuples for the default
     _steps_fitted_attr = "forecasters_"
 
-    def __init__(self, forecasters, n_jobs=None, fc_alt=None):
+    def __init__(
+        self, forecasters, n_jobs=None, fc_alt=None, backend=None, backend_params=None
+    ):
+        if n_jobs is not None:
```

Comment:
please add "todo" messages, deprecation instructions for release manager
https://www.sktime.net/en/stable/developer_guide/deprecation.html